### PR TITLE
Fix mobile quote img

### PR DESCRIPTION
**Pull request by Netaly79**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..1000;1,9..40,100..1000&display=swap" rel="stylesheet">
+  <!-- <link href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..1000;1,9..40,100..1000&display=swap" rel="stylesheet"> -->
```
Comment (saniola) [inline src/index.html]: ?????

**Review by saniola — APPROVED** [review]
everything is ok except fonts

File: src/index.html (hunk 1 above)
Comment (Netaly79) [reply src/index.html]: I will revert it, but this loading cost time. Maybe we can put font in fonts folder

Comment (saniola) [reply src/index.html]: Than as a separate PR

**Pull request merged**